app/models/spree/adjustment.rb

```diff
@@ -44,6 +44,8 @@ class Adjustment < ApplicationRecord
 
     belongs_to :tax_rate, -> { where spree_adjustments: { originator_type: 'Spree::TaxRate' } },
                foreign_key: 'originator_id'
+    belongs_to :voucher, -> { where spree_adjustments: { originator_type: 'Spree::Voucher' } },
```

Comment:
```suggestion
    belongs_to :voucher, -> { where spree_adjustments: { originator_type: 'Voucher' } },
```
This class isn't in the `Spree` namespace, right?

Reply:
Ah I see this builds on the previous PR, I'll move my review there... :+1: